tutorial/tests/testsuite/testsuite.py

```diff
@@ -237,6 +237,7 @@ def run_cell(self) -> List[IPytestResult]:
 
     @cell_magic
     def ipytest(self, line: str, cell: str):
+        print("Running tests...")
```

Comment:
Is it supposed to stay, or did you use that for testing?

Reply:
It was for testing, removed in c90d185